cfgov/unprocessed/css/atoms/overlay.less

```diff
@@ -0,0 +1,37 @@
+/* topdoc
+  name: Overlay
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <div class="a-overlay"></div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .a-overlay
+  tags:
+    - cfgov-atoms
+*/
+
+.a-overlay {
+    // Only show overlay at mobile/tablet size.
+    .respond-to-max( @bp-sm-max, {
+        height: 100%;
+        width: 100%;
+
+        position: fixed;
+        z-index: 10;
+
+        background: @black;
+        // Whitespace in content so element can have dimensions set.
+        content: '\a0';
```

Comment:
Interesting, I thought it wouldn't set the width and height without display:block or some content in the div, but I guess `position:fixed` takes care of it. Thanks!